src/components/pages/wallets/introduction.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Why is unsafe-assignment disabled here?

Reply:
@Stormtv shouldn't have been global, was disabling it when trying to get Lottie working as it's a bit wonky with how it does this. I've moved it to specific lines that it is needed